src/shared/varlink-io.systemd.Network.c

```diff
@@ -54,6 +71,10 @@ SD_VARLINK_DEFINE_INTERFACE(
                 &vl_method_GetNamespaceId,
                 &vl_method_GetLLDPNeighbors,
                 &vl_method_SetPersistentStorage,
+                SD_VARLINK_SYMBOL_COMMENT("Bring the specified link administratively down (clears IFF_UP)."),
```

Comment:
`administratively` ? It sounds ambiguous and provides no additional information. Please drop it.